pandas/tests/arrays/sparse/test_arithmetics.py

```diff
@@ -4,10 +4,44 @@
 import pytest
 
 import pandas as pd
+from pandas.core import ops
 from pandas.core.sparse.api import SparseDtype
 import pandas.util.testing as tm
 
 
+@pytest.fixture(params=["integer", "block"])
+def kind(request):
+    return request.param
```

Comment:
can you add a description here as a doc-string (just a 1-liner in quotes is good)